Source/missiles.cpp

```diff
@@ -562,7 +552,7 @@ bool MonsterMHit(int pnum, int m, int mindam, int maxdam, int dist, int t, bool
 	if (monster[m]._mmode == MM_CHARGE)
 		return false;
 
-	mor = monster[m].mMagicRes;
+	uint8_t mor = monster[m].mMagicRes;

```

Comment:
Isn't `mMagicRes` a uint16_t?

Reply:
```
MonsterStruct: uint16_t
UniqMonstStruct: uint16_t
MonsterData: uint8_t
```

It represents `monster_resistance` (`uint8_t`) as a bitflag.